src/pages/Browse/index.spec.tsx

```diff
@@ -114,14 +121,15 @@ describe(__filename, () => {
     const store = configureStore();
     store.dispatch(versionActions.loadVersionInfo({ version }));
 
+    // The user clicks a different file to view.
+    store.dispatch(
+      versionActions.updateSelectedPath({
```

Comment:
I'm a little unsure about this, but I might be wrong. Looking at the code for `Browse`, it never directly dispatches `updateSelectedPath`, but rather that is dispatched by the `fetchVersionFile` thunk. I suppose this may be the only way to test this case, but it just feels a bit odd to be dispatching this in a test for a component that never actually dispatches this. Is there a better way to do this, or do you think my concern is misplaced?

Reply:
I don't think there is because we want to test the case where the file is selected but there is no content for it yet, so we have to dispatch this action only.